docs/sponsorship/jobs.rst

```diff
@@ -0,0 +1,52 @@
+Job Posting Sponsorship
+=======================
+
+We have added a new sponsorship opportunity for companies looking to hire documentarians.
+It's a great way to promote your job to our community,
+which is full of folks that are passionate about software documentation,
+in particular around Docs as Code, API documentation, and developer experience.
+
+This sponsorship includes: 
+
+* A job posting in our monthly newsletter jobs section (Over {{ newsletter_subs }} subscribers)
+* A highlighted job posting in our ``#job-posts-only`` Slack channel ({{ slack_members }} members)
+
+Sponsored entry format
+----------------------
+
+The main goal here is to provide the company information, and a link to the job posting.
+The format is:
```

Comment:
This entry format is about putting your information front and centre, and putting the link to your job posting in front of our community.